packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -375,11 +517,18 @@ export const INVITE_PSM_COMMITTEE_MANIFEST = harden(
 /** @type {BootstrapManifest} */
 export const PSM_MANIFEST = {
   [makeAnchorAsset.name]: {
-    consume: { agoricNamesAdmin: true, bankManager: 'bank', zoe: 'zoe' },
+    vatParameters: { chainStorageEntries: true },
+    consume: {
+      agoricNamesAdmin: true,
+      bankManager: 'bank',
+      zoe: 'zoe',
+      anchorBalancePayments: true,
+    },
     installation: { consume: { mintHolder: 'zoe' } },
-    produce: { testFirstAnchorKit: true },
+    produce: { testFirstAnchorKit: true, anchorBalancePayments: true },
   },
   [startPSM.name]: {
```

Comment:
incidentally, `powersFor('startPSM')` isn't using this. please touch that up in boot-psm.

Reply:
I don't understand. `boot-psm.js` imports this same `PSM_MANIFEST`, and `allPowers` there includes `vatParameters`.

Reply:
```diff
--- a/packages/vats/src/core/boot-psm.js
+++ b/packages/vats/src/core/boot-psm.js
@@ -221,7 +221,7 @@ export const buildRootObject = (vatPowers, vatParameters) => {
         }),
       ),
       ...anchorAssets.map(anchorOptions =>
-        startPSM(powersFor('startPSM'), {
+        startPSM(powersFor(startPSM.name), {
           options: { anchorOptions },
         }),
       ),
```